ProcessLib/ProcessVariable.cpp

```diff
@@ -155,4 +136,19 @@ MeshLib::PropertyVector<double>& ProcessVariable::getOrCreateMeshProperty()
     }
 }
 
+std::vector<std::unique_ptr<BoundaryCondition>>
+ProcessVariable::getBoundaryConditions(
```

Comment:
I think the first one is sufficient. The caller doesn't need to know that there are underlying configs.